system-reports@eson57/system-reports@eson57.nemo_action.in

```diff
@@ -0,0 +1,7 @@
+[Nemo Action]
+_Name=System Reports
+Exec=mintreport
+Icon-Name=mintreport-symbolic
```

Comment:
These are only available on Mint system. Users of other distros will not be able to run this unless they compile the application manually. More [here](https://repology.org/project/mintreport/versions).

Reply:
Aha, I didn't think that far. I just thought that it was missing. Better close this PR then.
Thanks.

Reply:
It could complicate things depending on your goal but you could use something like `sosreport` (I don't think there's a gui) or something along those lines.

Reply:
No thanks, lets just skip it. I can use it internal.

Reply:
I will not leave mint anytime son.  ;)